core/trino-main/src/main/java/io/trino/operator/TableScanOperator.java

```diff
@@ -13,6 +13,7 @@
  */
```

Comment:
Add description in commit message why you do that

Reply:
Added